### ISSUE TITLE
[FEATURE] Добавить возможность отправки запросов без генерации тестов

### ISSUE THREAD

**Issue by ultras5631** [schemathesis/schemathesis#1275]
Что хочется получить : запуск утилиты по схеме свагера, создать запрос на каждый путь со всеми представленными для этого пути параметрами(из examples или подставленными рандомно), пропустить каждый запрос через прокси чтобы использовать в другой утилите ( burp suite )

Что есть на данный момент: Можно использовать 
--hypothesis-max-examples=1 --data-generation-method=negative

Но тогда подставляется очень малое кол-во параметров для каждого пути и покрытие получается не полным.

Если же ставить больше - тестов становится гораздо больше, а кол-во параметров все равно не полное.

**Comment by Stranger6667** [comment]
Т.е. в идеальной ситуации хотелось бы по одному запросу на каждый эндпоинт в которых присутствует максимум параметров / полей ?

**Comment by ultras5631** [comment]
> Т.е. в идеальной ситуации хотелось бы по одному запросу на каждый эндпоинт в которых присутствует максимум параметров / полей ?

Да, к примеру /getdate?id=1&datefrom=something&dateto=something&другойпараметр=значение вместо  
/getdate?id=1&id=0000&id=true&id=-123 и т.д.

Чтобы запросов было мало, но при этом они включали все доступные этому пути параметры

**Comment by Stranger6667** [comment]
Понятно, спасибо :)

Ниже немного заметок на эту тему (больше для меня на будущее)
Это можно было бы решить проверкой покрыта ли какая-то ветка в схеме (т.е. вызывать assume + опционально target) + модификацией схемы чтобы упростить задачу генератору.
